lang/Spanish.ini

```diff
@@ -381,6 +382,7 @@ MUSIC_VOLUME = "Volumen de Música"
 SFX_VOLUME = "Volumen de Efectos de Sonido"
 ENV_VOLUME = "Volumen de Entorno"
 FADEOUT = "Disminuir sonidos lejanos"
+MUTE_FOCUS_LOSS = "Silenciar el sonido cuando la ventana está desenfocada"
```

Comment:
Did you test to see if this fits in the panel?

Reply:
Retranslated, new text fits in panel